makefiles/tools/targets.inc.mk

```diff
@@ -17,10 +17,8 @@ $(RIOTTOOLS)/pic32prog/pic32prog: $(RIOTTOOLS)/pic32prog/Makefile
 	$(MAKE) -C $(@D)
 	@echo "[INFO] $(@F) binary successfully built!"
 
-$(RIOTTOOLS)/cc2538-bsl/cc2538-bsl.py:
-	@echo "[INFO] cc2538-bsl.py not found - fetching it from GitHub now"
-	CC= CFLAGS= $(MAKE) -C $(RIOTTOOLS)/cc2538-bsl
-	@echo "[INFO] cc2538-bsl.py successfully fetched!"
+$(RIOTTOOLS)/cc2538-bsl/cc2538-bsl.py: FORCE
```

Comment:
I'm not sure of this change. This makes it behave differently than the other tools. The issue with automatic fetch  when there's a version update is also there for other flashing tools (at least).

Reply:
You would rather leave at is?

Reply:
I could otherwise change the upstream repository while my patch is not accepted or merged? Your feedback on that patch is also welcomed.

Reply:
ping!

Reply:
> You would rather leave at is?

Yes. I know the local update is not automatic when the commit of the upstream project is changed but that's already the case for other flashing tools. I wouldn't do something different here but rather leave it as a follow-up.

Reply:
> Your feedback on that patch is also welcomed.

If it works it's ok. I don't want to review Python's code that should go to another repository.

Reply:
Ok, I'll remove the force and add the Makefile to the dependencies of the tool, that would force at lest this patch to be applied.